client/components/OrderHistory.js

```diff
@@ -14,28 +14,31 @@ class OrderHistory extends Component {
       return (
         <div>
           <h1>Previous Orders:</h1>
-          <ul>
+          <div>
             {orders.map((order) => {
               const date = new Date(order.createdAt);
               const readableDate = date.toDateString();
               return (
-                <li key={order.id}>
-                  <div>{readableDate}</div>
-                  <div>
-                    Total: $
+                <div key={order.id}>
+                  <div className="orderText">Order Date: {readableDate}</div>
+                  <div className="orderText">
+                    Item Cost: $
```

Comment:
minor point but can you call this Total: "instead of Item Cost"?

Reply:
updated!